### PR TITLE
feat: app metrics and axios optimization

### LINKED ISSUE #453: Outgoing requests metrics
### Please describe the problem that your request should resolve.
Implement measuring times of outgoing requests that can be enabled with an additional environmental variable. We need it to find out whats causing the SSR to sometimes respond very slowly.

### LINKED ISSUE #467: Fix performance issues on dev and production
### Context information
On dev and production, the application after some time gets slower. Restarting pods sometimes help. Investigate and fix the issue

### PR DESCRIPTION
**Pull request by lukeromanowicz**
## Description
Resolves #453 and Resolves #467 

The performance issues were caused by applying additional interceptors to the global axios object on every request

## Demo
image

The metrics cannot be seen by default unless somebody adds NUXT_PUBLIC_DEBUG_MODE to the environmental variables. Then the result will look like above

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] My change does not require a change to the documentation.

### CODE REVIEW

**Review by janmichek — APPROVED** [review]
Okay, cool 👍  Thanks for investigating and coming up with the fix. I didn't know the solution would cause such performance issues. I think it was not a problem on localhost, so I didn't suspecting this

File: src/stores/accountDetails.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { defineStore, storeToRefs } from 'pinia'
-import axios from 'axios'
 import { useRuntimeConfig } from 'nuxt/app'
+import useAxios from '@/composables/useAxios'
```
Comment (janmichek) [inline src/stores/accountDetails.js:3]: Would it be somehow possible to destruct the variable from it, so we can save one line per store?

**Review by michele-franchi — APPROVED** [review]
Nicee! Blazing fast 🚀

File: src/stores/accountDetails.js (hunk 1 above)
Comment (janmichek) [reply src/stores/accountDetails.js:3]: I see now, thanks for explanation

**Pull request merged**